SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -81,8 +81,10 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, QueuedIncomingRPCWaitTime(1.0f)
 	, QueuedIncomingRPCRetryTime(1.0f)
 	, QueuedOutgoingRPCRetryTime(1.0f)
-	, PositionUpdateFrequency(1.0f)
-	, PositionDistanceThreshold(100.0f) // 1m (100cm)
+	, PositionUpdateLowerThresholdSeconds(1.0f)       // 1 second
+	, PositionUpdateLowerThresholdCentimeters(100.0f) // 1m (100cm)
+	, PositionUpdateThresholdMaxSeconds(60.0f)      // 1 minute (60 seconds)
+	, PositionUpdateThresholdMaxCentimeters(5000.0f) // 50m (5000cm)
```

Comment:
Also considered naming these variables as following, but not sure if that would bring more clarity or make things worse, thoughts?
```
PositionUpdateNecessaryThresholdSeconds 
PositionUpdateNecessaryThresholdCentimeters
PositionUpdateSufficientThresholdSeconds 
PositionUpdateSufficientThresholdCentimeters
```